### ISSUE TITLE
ReSpec "Notes"

### ISSUE THREAD

**Issue by afs** [w3c/sparql-query#168]
The SPARQL documents have an overridding CSS definition for "note".

This CSS should be removed.

**Comment by afs** [comment]
Yes. It then becomes the same as the RDF documents which use the ReSpec definition. It would still be a box.

It's not widely used in SPARQL documents.

sparql-query/spec/index.html:4
sparql-results-csv-tsv/spec/index.html:1
sparql-results-json/spec/index.html:1
sparql-results-xml/spec/index.html:1
sparql-update/spec/index.html:8

The rest are 0.

I'll send in PR's for the 9 documents.

**Comment by Tpt** [comment]
Removal seems to have been done in all documents, closing the issue
